lib/video_editor/ffmpeg.py

```diff
@@ -152,9 +156,21 @@ def capture_thumbnail(self, stream_file, filename, duration, position):
                 position = duration - 0.1
             # create output file path
             output_file = f"{path_video}_preview_thumbnail.png"
+            cmd = ["-accurate_seek", "-i", path_video, "-ss", str(position), "-vframes", "1"]
+
+            vfilter = ''
+            if crop:
+                vfilter = f'-vf crop={crop["width"]}:{crop["height"]}:{crop["x"]}:{crop["y"]}'
+            if rotate:
+                vfilter += ',' if vfilter else '-vf '
+                transpose = f'transpose=1' if rotate > 0 else f'transpose=2'
+                vfilter += ','.join([transpose] * (rotate // 90))
+                print(vfilter)
+            if vfilter:
+                cmd.extend(vfilter.split(' '))
+
             # run ffmpeg command
-            subprocess.run(["ffmpeg", "-v", "error", "-y", "-accurate_seek", "-i", path_video,
-                            "-ss", str(position), "-vframes", "1", output_file])
+            subprocess.run(["ffmpeg", "-v", "error", "-y", *cmd, output_file])
```

Comment:
I think we can update (if needed) and reuse `_run_ffmpeg` method here.

Reply:
We can't reuse `_run_ffmpeg` because of `-accurate_seek` must be put before input file. So should we add optional param before input file like: `ffmpeg *optional_param* -i input_file`.

Reply:
that sounds ok